src/templates/layouts/healthCareLocalFacility/Phone.test.tsx

```diff
@@ -22,7 +23,7 @@ describe('Phone', () => {
   it('renders fieldTelephone info', () => {
     const { container } = render(
       <Phone
-        fieldTelephone={{
+        mentalHealthPhoneNumber={formatPhone({
```

Comment:
I guess I could have just made an object literal with the result of this function, but I was being lazy. 🙈